pyproject.toml

```
@@ -0,0 +1,42 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=60",
+    "setuptools-scm>=8.0"
+    ]
+
+[project]
+name = "aiu-fms-testing-utils"
+dynamic = ["version"]
+description = "Spyre FMS Testing Utils"
+authors = [
+  {name = "Thara Palanivel", email = "Tharangini.Palanivel@ibm.com"},
+  {name = "Ted Chang", email = "htchang@us.ibm.com"},
+]
+license = "Apache-2.0"
+readme = "README.md"
+requires-python = "~=3.12"
+keywords = ['aiu-fms-testing-utils', 'python', 'utils']
+classifiers=[
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12"
+]
+dependencies = [
+"ibm-fms @ git+https://github.com/foundation-model-stack/foundation-model-stack@main",
+"fms-model-optimizer @ git+https://github.com/foundation-model-stack/fms-model-optimizer@main",
```

test pypi rejected direct GitHub dependencies
```suggestion
"ibm-fms",
"fms-model-optimizer",
```

---

Ah, great suggestion - ok - just tried it and it looks good - will modify.